packages/client/codegen-api.yml

```diff
@@ -24,12 +24,12 @@ config:
     FollowModuleData: string
     Handle: string
     HandleClaimIdScalar: string
-    IpfsCid: string
+    ImageSizeTransform: string
```

Comment:
Can we copy over from react package the more type safe version of `ImageSizeTransform`?

Reply:
ok, I added it for both api-bindings and client packages 

https://github.com/lens-protocol/lens-sdk/pull/424/commits/ebfa1011e39f64501eed074f2780792ed067d782